prac_07/project_management.py

```diff
@@ -0,0 +1,87 @@
+"""
+CP1404/CP5632 Practical
+Do-from-scratch - Project Management Program
+Estimate: minutes
+Actual: minutes
+"""
+from project import Project
+
+
+def main():
+    projects = []
+    print_menu()
+    choice = input(">>> ").upper()
+    while choice != "Q":
+        # if choice == "L":
+        # in_filename = input("Data file to load from: ")
+        projects = load_data("projects.txt", projects)
+        # print("Data loaded.")
+        if choice == "S":
+            out_filename = input("Data file to save to: ")
+            write_data(out_filename, projects)
+            print("Data saved.")
+        if choice == "D":
+            display_projects(projects)
+        if choice == "U":
+            for project in projects:
+                print(f"{projects.index(project)} {project}")
+            project_choice = input("Project Choice: ")
+            print(projects[project_choice])
+            new_percentage = int(input("New Percentage: "))
+        print_menu()
+        choice = input(">>> ").upper()
+
+
+def display_projects(projects):
+    print("Incomplete projects:")
+    incomplete_projects = get_incomplete_projects(projects)
```

Comment:
Nothing wrong with calling another function inside this function
But do we really need a function just for a list comprehension?
Especially if this is the only time you call the function, I suggest just putting the list comprehension inside the display project function :)